atomic_reactor/plugins/pre_koji_parent.py

```diff
@@ -74,8 +75,17 @@ def detect_parent_image_nvr(self):
         config = self.workflow.base_image_inspect[INSPECT_CONFIG]
         labels = config['Labels'] or {}
 
+        label_names = 'com.redhat.component', 'version', 'release'
+        for label_name in label_names:
+            if label_name not in labels:
+                self._parent_image_nvr = None
+                self.log.info("Failed to find label '%s' in parent image. "
+                              "Not waiting for Koji build." % label_name)
```

Comment:
landscape.io suggests passing `label_name` as a separate parameter to the format string, rather than performing the formatting before the call, e.g. `self.log.info("Failed to ... %s ...", label_name)`.